packages/pluggableWidgets/maps-web/src/components/GoogleMap.tsx

```diff
@@ -117,7 +121,9 @@ export function GoogleMap(props: GoogleMapsProps): ReactElement {
                                 />
                             ))}
                     </GoogleMapComponent>
-                </LoadScript>
+                ) : (
+                    <div className="spinner" />
```

Comment:
do we have this class somewhere?

Reply:
what do you mean? I copied this over from te previous `loadingElement`

Reply:
ok